src/main/java/training/subsystems/wrist/Wrist.java

```diff
@@ -0,0 +1,53 @@
+package training.subsystems.wrist;
+
+import com.revrobotics.CANSparkBase;
+import com.revrobotics.CANSparkLowLevel;
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.math.geometry.Rotation2d;
+import utils.GBSubsystem;
+
+public class Wrist extends GBSubsystem {
+
+	private static Wrist instance;
+	private final CANSparkMax motor;
+
+	private Wrist() {
+		motor = new CANSparkMax(WristConstant.MOTOR_ID, CANSparkLowLevel.MotorType.kBrushless);
+		motor.getPIDController().setP(WristConstant.P);
+		motor.getPIDController().setI(WristConstant.I);
+		motor.getPIDController().setD(WristConstant.D);
+	}
+
+	public static Wrist getInstance() {
+		init();
+		return instance;
+	}
+
+	public static void init() {
+		if (instance == null) {
+			instance = new Wrist();
+		}
+	}
+
+	public void goToAngle(Rotation2d targetAngle) {
+		motor.getPIDController().setReference(targetAngle.getDegrees(), CANSparkBase.ControlType.kPosition);
```

Comment:
CANSparkBase.ControlType.kPosition is a constant, same in the other classes where you used it

Reply:
no, it's ok. its a constant for what control type to use, she doesn't need to create a constant for Rev's constant